fvm/src/kernel/blocks.rs

```diff
@@ -61,19 +65,42 @@ impl Block {
 }
 
 #[derive(Error, Debug)]
-pub enum BlockError {
-    #[error("block {0} is unreachable")]
-    Unreachable(Box<Cid>),
+pub enum BlockPutError {
```

Comment:
I'm tempted to just use syscall errors, but technically this isn't a part of the syscall interface.